File: SGLKitTest/Document.m
//
//  Document.mm
//  SGLKitTest
//
//  Created by Colin Caufield on 2014-11-14.
//  Copyright (c) 2014 Secret Geometry. All rights reserved.
//

#import "Document.h"
#import "WindowController.h"


@implementation Document

- (void) makeWindowControllers
{
    self.windowController = [[WindowController alloc] init];
    [self addWindowController:self.windowController];
}

@end

File: SGLKitTestSwift/Document.swift
//
//  Document.swift
//  SGLKitTestSwift
//
//  Created by Colin Caufield on 2016-01-31.
//  Copyright © 2016 Secret Geometry, Inc. All rights reserved.
//

import Cocoa

class Document: NSDocument {
    
    var windowController: WindowController!
    
    override func makeWindowControllers() {
        self.windowController = WindowController()
        self.addWindowController(self.windowController)
    }
}
